src/frontend/helpers/library.ts

```diff
@@ -53,7 +53,7 @@ async function install({
     if (!folder_name) return
     return handleStopInstallation(
       appName,
-      [installPath, folder_name],
+      installPath,
```

Comment:
we don't need the folder_name here, we can get this info when stopping the download in the backend, it simplifies the pause/cancel calls